client/client.go

```diff
@@ -140,6 +140,11 @@ func (c *Client) On(eventType EventType, h EventHandler) {
 	c.handlers[eventType] = h
 }
 
+// SendWs sends a websocket event to the server
+func (c *Client) SendWs(ev string, msg any, binary bool) error {
+	return c.wsSend(ev, msg, binary)
+}
```

Comment:
Wondering if we should just unexport the other one. We are essentially wrapping with no change whatsoever.
Also `SendWS` would be better for my OCD.

Reply:
Your OCD is interesting, cause there are other methods with `wsVerb` below. :)

Reply:
@cpoile OCD was on proper camel casing :p `WS` vs `Ws`

Reply:
@streamer45  ahhh. :P  Well, now I'm in a bind, because does `WSSend` work, all uppercase? or `WsSend` (ocd-unfriendly camel casing)

Reply:
stop it :p